tools/markdown-magic/package.json

```diff
@@ -0,0 +1,41 @@
+{
+	"name": "@fluid-tools/markdown-magic",
+	"version": "0.1.0",
```

Comment:
Any reason not to startt at 1.0? Semver is easier to reason about post 1.0.

Reply:
No particular reason, no. I can update if you would prefer, though I do sort of like how pre-1.0.0 sort of implies half-baked-ness, which I think applies here, haha.